CMSgov/mint-app
# [EASI-2222] Read-only Beneficiaries - FE

# EASI-2222

## Changes and Description

- Creates new Beneficiaries Read Only component
- Add new `GetAllBeneficiaries` query and type
- Add `translateTriStateAnswer` to `src/utils/modelPlan.ts`
- Adds test and snaps 


## AC
- Render questions and answers from the task list section
- Should still render unanswered questions - populated with 'No answer entered'
- Testing

## How to test this change
1. Navigate to the Model Summary ( Models on nav > Complete Plan > View this Model Plan on sidebar)
2. Click `Beneficiaries` on sidebar
3. Ensure component is displayed correctly and matches figma

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
